warrior/Tools/reporting/html_results_template.html

```diff
@@ -622,20 +660,17 @@ <h2>Results Summary</h2>
 		<tr id="tr1" name="HeadingRow">
 			<th>Type</th>
 			<th>Name</th>
-			<th>Info</th>
+		        <th>Info</th>
+			<th>Description</th>
 			<th>Timestamp</th>
 			<th>Duration</th>
 			<th>Status</th>
 			<th>Impact</th>
-			<th>On Error</th>
-			<th>
-			<span style="padding-left:5px">Results (<i class="fa fa-line-chart">)</i></span><br/>
-			<span style="padding-left:5px">Logs (<i class="fa fa fa-book">)</i></span><br/>
-			<span style="padding-left:5px">Defects (<i class="fa fa fa-bug">)</i></span><br/>
-			</th>
-			<th colspan="6" id="summary">Summary</th>
-		</tr>
-
+            		<th>On Error</th>
+           		<th>Results (<i class="fa fa-line-chart">)</th>
```

Comment:
We do not need the icons inside the parenthesis now that the results, defects, logs columns have been separated. The icon is also causing the "Results" header to overflow - removing the icons would fix that issue.

Reply:
@sanikakulkarni 
Removed the icons for result, log & defect in table header of HTML Result Summary.